### ISSUE TITLE
Add binary search algorithm

### ISSUE THREAD

**Issue by ankit-kmar** [ankit-kmar/Data-Science-with-Python#1]

**Comment by anjali-singh16** [comment]
assign me the doc part.
@ankit-kmar

**Comment by PrasadKishorDesai** [comment]
please assign this work to me, I will definitely do it properly

**Comment by Chayani-Basak** [comment]
Please assign me the documentation part of this.

**Comment by ankit-kmar** [comment]
> Please assign me the documentation part of this.
This issue has been taken. You may take its video part or can add some another algorithm.

**Comment by prakashr7d** [comment]
Any other things to be added, if any assign me i am happy to do!!

**Comment by ankit-kmar** [comment]
> Any other things to be added, if any assign me i am happy to do!!

yes please you can create the issue on which you want to work. @prakashr7d

**Comment by chinmayilondhe** [comment]
@ankit-kmar Please assign me the video part
